repositories.yaml

```diff
@@ -288,6 +304,7 @@ applications:
       - org.mozilla.components:lib-crash
       - org.mozilla.components:support-sync-telemetry
       - org.mozilla.appservices:logins
+      - org.mozilla.components:support-migration
```

Comment:
dependencies need to be inclusive of history, or pings from dependencies will be deleted from schemas, which should only be done if we want the ping table deleted.

Reply:
Thanks, this is good to know.